.prospector.yaml

```diff
@@ -1,5 +1,6 @@
 output-format: text
 
+autodetect: false
```

Comment:
Prospector seemed to think we were using Django, but then complaining that we didn't have a Django library...